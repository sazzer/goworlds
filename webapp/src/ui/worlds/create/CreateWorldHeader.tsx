import React, {FunctionComponent} from 'react';
import {Header} from "semantic-ui-react";
import {useTranslation} from "react-i18next";

/** The type for the header */
export declare type CreateWorldHeaderProps = {
};

/**
 * Header to render on the CreateWorld page
 * @constructor
 */
export const CreateWorldHeader: FunctionComponent<CreateWorldHeaderProps> = () => {
    const { t } = useTranslation();

    return <Header size='large' dividing>
        {t('worlds.create.title')}
    </Header>;
}
