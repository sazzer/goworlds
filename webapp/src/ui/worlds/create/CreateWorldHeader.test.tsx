import React from 'react';
import {CreateWorldHeader} from './CreateWorldHeader';
import {render} from '@testing-library/react';

/** Set up the component to test */
function setup() {
    return {
        element: render(<CreateWorldHeader />)
    };
}

it('renders without crashing', () => {
    const {element} = setup();

    expect(element.baseElement).toMatchSnapshot();
});
