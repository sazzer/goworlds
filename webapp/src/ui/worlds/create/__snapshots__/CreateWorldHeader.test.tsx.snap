// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
<body>
  <div>
    <div
      class="ui large dividing header"
    >
      Create New World
    </div>
  </div>
</body>
`;
